docs/_documentations/importing-existing-projects.md

```diff
@@ -15,24 +15,33 @@ Configure a project to run in Codewind.
 ## Importing projects from IDEs
 
 Add a project from an existing Git repository:
+
 1. Clone the repository to your local computer.
 2. Use the **Add Existing Project** command to add the project to Codewind.
 
 ## Requirements for importing an existing project not created in Codewind or through Appsody or Odo
 
-Local projects are built and run on Docker. To import your project to a local Codewind instance, it must include a Dockerfile. For more information about Dockerfiles, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/).
+Local projects are built and run on Docker. To import your project to a local Codewind instance, you must include a Dockerfile. For more information about Dockerfiles, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/).
+**Note:** The Dockerfile must contain an `EXPOSE` instruction to point to the port that the project runs on.
+
+Remote projects are built and run on Kubernetes. To import your project to a remote Codewind instance, you must include a Dockerfile and a Helm chart. For more information about Dockerfiles and Helm charts, see: 
 
-Remote projects are built and run on Kubernetes. To import your project to a remote Codewind instance, it must include a Dockerfile and a Helm chart. For more information about Dockerfiles, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/) and for more information about Helm charts, see [Helm chart documentation](https://helm.sh/docs/topics/charts/).
+* [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/)
+* [Helm chart documentation](https://helm.sh/docs/topics/charts/).
```

Comment:
No period needed at the end of the bullet point.

Reply:
Or put it on one line: "...and Helm charts, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/) and [Helm chart documentation](https://helm.sh/docs/topics/charts/)."

I kind of prefer this way.